src/client/driver/command-executors/execute-action-command.js

```diff
@@ -100,7 +101,7 @@ function ensureCommandElements (command, timeout) {
     var ensureElementPromises = [];
 
     if (command.selector) {
-        ensureElementPromises.push(ensureElement(command.selector, timeout,
+        ensureElementPromises.push(ensureElement(command.selector, command.elementAvailabilityTimeout || timeout,
```

Comment:
I guess `wait` is not an `action`, so we can create separate executor for it and refactor `ensureElement` into separate module